### DISCUSSION TITLE
how to install moleculekit

### DISCUSSION THREAD

**Discussion by weishipitt** [torchmd/torchmd#25] [General]
Hi, good morning! 

By following the "Installation" part (https://github.com/torchmd/torchmd) and using the conda, I have created an environment variable torchmd and the following instructions to install torchmd. "conda create -n torchmd
conda activate torchmd
conda install pytorch torchvision cudatoolkit=10.1 -c pytorch
conda install ipython

pip install torchmd"

My question is how to install moleculekit using conda under this torchmd environment. The "conda install -c acellera   moleculekit" does not work and it says something like moleculekit can not be found in acellera. 

Thanks for your help,

Wei Shi

**Comment by stefdoerr** [comment]
```
conda install -c acellera -c conda-forge moleculekit
```
should work. If it doesn't paste the full error here please.

**Reply by weishipitt** [reply]
It still get the following error message. Note that I am trying to install this torchmd on my laptop (Windows). 

(torchmd) PS C:\Users\weish> conda install -c acellera -c conda-forge moleculekit
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - moleculekit

Current channels:

  - https://conda.anaconda.org/acellera/win-64
  - https://conda.anaconda.org/acellera/noarch
  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

**Reply by stefdoerr** [reply]
There is no version of our software for Windows. We only support Linux platforms

**Reply by stefdoerr** [reply]
If you feel brave however you can try `pip install moleculekit` which might work. But it's very untested

**Reply by weishipitt** [reply]
I will try to install the torchmd on the super-computer, which is running Unix.
